src/layman/upgrade/upgrade_v1_16.py

```diff
@@ -29,6 +36,32 @@ def adjust_db_publication_layer_srid_data():
     db_util.run_statement(statement, (srid, LAYER_TYPE))
 
 
+def adjust_maps():
+    logger.info(f'    Adjust maps')
+    query = f'''select w.name,
+        p.name
+    from {DB_SCHEMA}.publications p inner join
+         {DB_SCHEMA}.workspaces w on w.id = p.id_workspace
+    where p.type = %s
+    ;'''
+    publications = db_util.run_query(query, (MAP_TYPE, ))
+    for workspace, publication in publications:
+        map_file_path = input_file.get_map_file(workspace, publication)
+        mapjson = map_util.get_map_file_json(workspace, publication)
+        bbox_json = map_util.get_bbox_from_json(mapjson)
+        crs = map_util.get_crs_from_json(mapjson)
+        bbox = bbox_util.transform(bbox_json, JSON_EXTENT_CRS, crs)
+        db_publications.set_bbox(workspace, MAP_TYPE, publication, bbox, crs, )
+
+        mapjson['describedBy'] = "https://raw.githubusercontent.com/hslayers/map-compositions/2.0.0/schema.json"
+        mapjson['schema_version'] = "2.0.0"
+        mapjson['extent'] = bbox_json
+        mapjson['nativeExtent'] = bbox
+
```

Comment:
I would like to validate each migrated map JSON to be sure it's valid against v2.0.0. If not, it should raise an error and migration should stop.